### DISCUSSION TITLE
Upload File

### DISCUSSION THREAD

**Discussion by hairil2912** [nasirkhan/laravel-starter#496] [Q&A]
How do I create a form to upload specific files other than images, for example, PDFs? 🙏

**Comment by nasirkhan** [comment]
Uploading an image and other files are the same. You have to change the file validation from your controller. 
In this Laravel Starter project, I used [spatie/laravel-medialibrary](https://spatie.be/docs/laravel-medialibrary/v10/introduction) package to manage file upload. You may follow their doc or just use the [default Laravel approach](https://laravel.com/docs/10.x/filesystem#file-uploads) to upload files.
